components/ArticleReplySummary.js

```diff
@@ -38,9 +43,23 @@ function ArticleReplySummary({ articleReply, className, ...props }) {
     </ProfileTooltip>
   );
 
+  useEffect(() => {
+    if (anchorRef.current.hash === location.hash) {
+      anchorRef.current.scrollIntoView({ behavior: 'smooth' });
+    }
+  }, []);
+
   return (
     <div className={cx(classes.replyType, className)} {...props}>
       {jt`${authorElem} mark this message ${TYPE_NAME[replyType]}`}
+      <a
+        ref={anchorRef}
+        className={classes.hash}
+        href={`#${articleReply.replyId}`}
+        onClick={e => e.target.scrollIntoView({ behavior: 'smooth' })}
```

Comment:
`scrollIntoView` is probably not needed if we define a global scroll-behavior in CSS.

However, it would be nice if clicking `#` copies the URL + href into clipboard. We are already using `copy-to-clipboard` package.